### PR TITLE
setting deployment through sftp on travis and appveyor

### PR DESCRIPTION
**Pull request by eolivi**
- CGAL is not included since the executable are too big for a release (TODO check that it is normal). 
- only send through sftp if ENABLE_PACKAGING is ON and on master branch
- private ssh rsa key aes256 encoded check openmeeg-admins@inria.ffr email for clarifications

### CODE REVIEW

**Comment by agramfort** [comment]
working?

**Comment by eolivi** [comment]
yes finally working !
it has uploaded this one http://openmeeg.gforge.inria.fr/download/OpenMEEG-2.3.0.2-Linux-Auto-static.tar.gz 
but I have set conditions, so that id deploys only if we enable packaging AND we are on master branch.

**Comment by agramfort** [comment]
great !

**Comment by eolivi** [comment]
P.S:  I don't get why codecov complains ...

**Comment by agramfort** [comment]
no idea

regarding naming "Auto" is maybe not explicit right?

**Comment by eolivi** [comment]
you are right. I've changed that in my next PR (the MKL one).

**Comment by eolivi** [comment]
do not merge yet, as we will probably go for a common solution (with your python script) on both travis + appveyor

**Comment by agramfort** [comment]
can you tell me when I can test the upload binaries? We'll also ask Francois to test for us with brainstorm

File: .appveyor.yml [hunk 1]
```diff
@@ -33,23 +45,30 @@ build_script:
...
     # needed to find the dlls (VTK, (OpenBLAS, mingw), MKL)
   - setPATH.cmd
-  - echo %PATH%
-  - ctest -V
+  - ctest
+
+on_success:
+  - set src_dir=%APPVEYOR_BUILD_FOLDER%
+  - cd %src_dir%
+  - dir
```
Comment (eolivi) [inline .appveyor.yml]: I can remove this 'dir' command

**Comment by eolivi** [comment]
Everything seem OK now.
This PR is done from the branch deploy_gforge
- one appveyor failed because you cannot use 'secure variable' from a PR (security reasons)
- the codecov complains, because, I disabled CGAL for the build which is doing the codecov.

**Comment by agramfort** [comment]
cool

does it mean that appveyor will fail on a PRs now?

**Comment by eolivi** [comment]
you are right ! I'll put a condition on PR so that this does not happen. 
anything else to say ?

**Comment by agramfort** [comment]
nope. Besides looks good

**Comment by eolivi** [comment]
arffff I wrongly changed the commit message... I'll submit again ...

**Pull request closed without merging**